src/ring_theory/adjoin/basic.lean

```diff
@@ -205,6 +206,12 @@ begin
   congr' 1 with z, simp [submonoid.closure_union, submonoid.mem_sup, set.mem_mul]
 end
 
+variable (A)
+
+lemma adjoin_le_restrict_scalars (B : Type*) [semiring B] [algebra R B] [algebra A B]
+  [is_scalar_tower R A B] (s : set B) : adjoin R s ≤  subalgebra.restrict_scalars R (adjoin A s) :=
```

Comment:
Is it really worth even stating this lemma here, at the cost of the extra import? The proof is barely longer than the theorem name.